README.md

```diff
@@ -1,2 +1,40 @@
 # tubeclone
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
 Our first Project which is to clone Youtube website 
+
+#Youtube's video player page
+The goal of this project is to focus on embedding the youtube video player into the page so it can actually play and show little thumbnail images along the right side.
+
+![screenshot](https://user-images.githubusercontent.com/30318155/90988765-94f20c00-e595-11ea-9f83-dccee4ec7011.png)
+
+## Technologies
+-Built HTML5
+--We used the header  tag to contain the logos and all the top nav elements
+--We used the flex to control the elements and items in the header
+--We mostly used the divs eleent to structure the contents of the main page 
+--We created an aside main element for the thumbnails and right side images and iformation
+we also used the youtube iframe to embedd the youtube video player.
+-CSS main used pure CSS3 and the flex box for ourpage design
+
+-youtube iframe API
+
+Live Link https://rawcdn.githack.com/tahiry-dev/tubeclone/e8622d2fb2feedfa7f25d177144ae15d47d15fb5/index.html
+
+### Prerequisites
+A browser that supports CSS, FLEX and Grid
+You can click on the link to view a live demo of the site.
+
+**Author1**
+https://github.com/tnyandoro
+https://www.linkedin.com/in/tendai-nyandoro-a8060826/
```

Comment:
- [ ] You can display these lines correctly by adding a black line between the text OR by making a list with bullet points, just use this sintax: `* Your text`. Please make sure to check your preview of the .md files to make sure they are displaying correctly